src/twisted/names/test/test_names.py

```diff
@@ -31,8 +31,8 @@
 from twisted.trial import unittest
 
 
-def justPayload(results):
-    return [r.payload for r in results[0]]
+def justAnswerPayload(resolver_response):
+    return [r.payload for r in resolver_response.answer]
```

Comment:
```suggestion
def justAnswerPayload(resolverResponse):
    return [r.payload for r in resolverResponse.answer]
```

Reply:
Would you mind adding a quick docstring for this given that the logic is changing a bit here?